tests/test-libarchive-import.c

```diff
@@ -453,6 +456,90 @@ test_libarchive_xattr_callback (gconstpointer data)
   g_assert_no_error (error);
 }
 
+static void
+test_libarchive_xattr_import (gconstpointer data)
+{
+  TestData *td = (void*)data;
+  GError *error = NULL;
+  g_autoptr(OtAutoArchiveRead) a = archive_read_new ();
+  OstreeRepoImportArchiveOptions opts = { 0 };
+  char buf[15] = { 0 };
+
+  if (skip_if_no_xattr (td))
+    goto out;
+
+  if (td->skip_all != NULL)
+    {
+      g_test_skip (td->skip_all->message);
+      goto out;
+    }
+
+  test_archive_setup (td->fd, a);
+
+  opts.ignore_unsupported_content = TRUE;
+
+  if (!import_write_and_ref (td->repo, &opts, a, "baz", NULL, &error))
+    goto out;
+
+  /* check contents */
+  if (!spawn_cmdline ("ostree --repo=repo checkout baz import-checkout", NULL))
+    goto out;
+
+  spawn_cmdline ("ostree --repo=repo ls -R -X baz", NULL);
+  spawn_cmdline ("attr -l import-checkout/anotherfile", NULL);
```

Comment:
Left-over from debugging? Or did you mean to leave this around for future debugging as well?

Reply:
Leaving it in was intentional.  I've got some plans to adjust these tests in the future.